.github/workflows/ios_e2e_tests.yml

```diff
@@ -0,0 +1,60 @@
+name: iOS
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: macos-10.15
+
+    env:
+      DEVELOPER_DIR: /Applications/Xcode_12.4.app
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+        
+    - name: Create env file
+      run: |
+        touch .env
+        echo API_URL=${{ secrets.API_URL }} >> .env
```

Comment:
Are you using the production API for your testing ?

Reply:
Yes, I'm using this one 'https://api.safely-app.fr', is that okay ?

Reply:
Yes perfect :+1: